tests/gold_tests/pluginTest/url_sig/url_sig.test.py

```diff
@@ -63,30 +63,31 @@
 
 # Use unchanged incoming URL.
 #
+ts.Setup.Copy("url_sig.config", ts.Variables.CONFIGDIR)
 ts.Disk.remap_config.AddLine(
     'map http://one.two.three/ http://127.0.0.1:{}/'.format(server.Variables.Port) +
-    ' @plugin=url_sig.so @pparam={}/url_sig.config'.format(Test.TestDirectory)
```

Comment:
just a general note... This could be clean up more via saying 
`f'map http://one.two.three/ http://127.0.0.1:{server.Variables.Port}/' +`

This can be done to a number of cases below

Reply:
Yes, I always use this for new code. But I kept the method consistent with the rest of the file which generally uses the older syntax.

Reply:
Actually, for this file it unambiguously improves the file. I'll apply it throughout.